pyroomacoustics/bss/ilrma.py

```diff
@@ -62,72 +89,97 @@ def ilrma(X, n_src=None, n_iter=20, proj_back=False, W0=None,
         W = W0.copy()
 
     # initialize the nonnegative matrixes with random values
-    T = np.array(np.random.rand(n_freq, n_components, n_src))
-    V = np.array(np.random.rand(n_components, n_frames, n_src))
-    Y = np.zeros((n_frames, n_freq, n_src), dtype=X.dtype)
-    R = np.zeros((n_freq, n_frames, n_src))
+    T = np.array(0.1 + 0.9 * np.random.rand(n_src, n_freq, n_components))
+    V = np.array(0.1 + 0.9 * np.random.rand(n_src, n_frames, n_components))
+    R = np.zeros((n_src, n_freq, n_frames))
     I = np.eye(n_src, n_src)
     U = np.zeros((n_freq, n_src, n_chan, n_chan), dtype=X.dtype)
     product = np.zeros((n_freq, n_chan, n_chan), dtype=X.dtype)
     lambda_aux = np.zeros(n_src)
-    machine_epsilon = np.finfo(float).eps
+    machine_epsilon = 1e-15
```

Comment:
I think we should call this parameter the same across algorithms, i.e., either `machine_epsilon` or `eps`. For `auxiva`, I would define it in the beginning as well as we do it here.

Reply:
I have standardized to `eps` because `machine_epsilon` has a different meaning (the smallest float that can be represented).